docs/source/guide/configuration.rst

```diff
@@ -48,9 +50,76 @@ To set these configuration options, create a ``Config`` object with the options
 
     client = boto3.client('kinesis', config=my_config)
 
+
+Using proxies
+~~~~~~~~~~~~~
+Boto3 supports using proxies as intermediaries between your code and AWS. Proxies may provide any number of functions, from filtering to security and firewalls to privacy assurance.
```

Comment:
With Boto3, you can use proxies as...

Proxies can provide functions such as filtering, security, firewalls, and privacy assurance.